### DISCUSSION TITLE
Windows runtime requirements

### DISCUSSION THREAD

**Discussion by moogstuart** [libpnet/libpnet#577] [Q&A]
Hi, I see in the readme that to build on windows requires WinPcap or npcap to be installed as well as a file to be placed in a particular location. Is this only needed for the build or is this also a runtime requirement?
Thanks in advance for any answers.

**Comment by stappersg** [comment]
@moogstuart consider reporting your findings.

**Reply by moogstuart** [reply]
@stappersg I will if/when I get around to testing it.... I was hoping someone else might already know.

**Reply by stappersg** [reply]
Probably more windows users are hoping that.

**Comment by Akmot9** [comment]
What do you think about the issue i opened about creating a documentation for windows user ? : https://github.com/libpnet/libpnet/issues/646

**Reply by stappersg** [reply]
It is not important what others think, much better are _merge requests_, people who _do_.

**Reply by Akmot9** [reply]
https://github.com/libpnet/libpnet/wiki/Windows-10-compilation

I made a wiki page then.
